src/Model/Post.php

```diff
@@ -537,6 +538,19 @@ protected function init() {
 
 					return false;
 				},
+				'isPostsPage'              => function ()
```

Comment:
formatting is off. 

I probably should automate this to fix on PRs, but for now you can do the following:

```
composer install
composer run fix-cs
composer install --no-dev
```

And that should fix the formatting. Then push the changes.

Reply:
ran fix-cs and pushed changes